### DISCUSSION TITLE
2024-01-25 중간 회고

### DISCUSSION THREAD

**Discussion by kses1010** [code-itch/effective-typescript#41] [General]
### 앞으로 어떻게 하면 좋을지

**Comment by gw-lim** [comment]
## 스터디 하면서 좋았던 점
- 각자 하나의 챕터를 맡아서 분배를 하다보니 바쁜 주에는 모든 챕터를 읽지 않아도 다른 스터디원분들의 발표를 통해 해당 챕터의 내용을 알 수 있어 좋았습니다.
- 깃허브에 각자 정리한 내용을 올리다보니 나중에 다시 찾아보기 좋았어요.
- 다른 분들이 책 내용말고 추가로 조사한 내용들을 설명해주실 때 좋았어요.
- 스터디원들이 좋았다~

## 아쉬웠던 점
- 점점 배정된 주제만 읽고 다른 분들의 주제는 읽지 못하다보니 추가적인 질문을 못하게 된 것 같아 스스로에게 아쉽습니다.

## 앞으로 어떻게 하면 좋을지
- 지금처럼만 쭉 하면 너무 좋을 것 같아요~

**Comment by BeMatthewsong** [comment]
## 스터디 하면서 좋았던 점
- 발표하기 위해 본인이 맡은 주제를 쉽게 표현하려는 글들이 많이 보여 좋습니다.
- 확실히 책보다 스터디를 통해 개념을 정립하는 과정이 좋습니다.
- 상희님이 진행을 잘해주시려는 모습이 팀원으로서 감사함을 느낍니다. (진행자의 중요성 👍🏻)
- 확실히 팀원들의 반응이 좋습니다.  그래서 더 편안한 분위기를 조성하는 것 같아요.
- 지금 같은 진행 속도가 적절하다고 생각합니다.

## 아쉬웠던 점
- 점점 일정에 치여 다른 사람의 주제를 자세히 못 살피는 경우가 생겨서 제 스스로가 아쉽습니다.

## 앞으로 어떻게 하면 좋을지
- 지금처럼 특정 주제에 대해 처음 보거나 잠시 잊은 사람들을 위해 예제를 통해 쉽게 설명하는 방향으로 했으면 합니다.

**Comment by kses1010** [comment]
### 스터디 하면서 좋았던 점
- 사실 크게 재밌지 않은 이펙티브 타입스크립트를 억지로라도 읽어가면서 진행해서 좋았습니다.
- 나혼자 읽었다면 몰랐을 지식이 스터디하면서 하나씩 늘어나서 좋았습니다.
- 발표를 해야하니 책에 있는 지식뿐만아니라 다른 정보도 제공하고 싶어 추가적으로 찾아다니는 모습을 발견할 수 있었습니다.
### 아쉬웠던 점
- 갈수록 나의 주제만 공부하고 다른 사람의 주제에 대해서 질문을 하지 못한게 아쉽습니다.

### 앞으로 어떻게 하면 좋을지
- 체계적으로 다른사람 주제에 관심을 가지게 하는 방법을 찾는게 우선이라고 생각합니다.

**Comment by qooktree1** [comment]
### 스터디 하면서 좋았던 점
- 책을 혼자 읽으면서 이해되지 않았던 부분들이나 이해를 하는데 오랜 시간이 걸렸던 부분들을 스터디를 통해 배울 수 있는 점이 좋았습니다.
- github에 정리를 하며 스터디를 진행해 나중에 모르는 부분을 찾아보거나 기록하는 데에 있어서 이점을  주는 것 같아 좋았습니다. 👍👍
- 혼자 공부하는게 아니다 보니 강제성이 생겨 의욕이 생기는 것 같아 좋았습니다. 👍

### 아쉬웠던 점
- 자신의 주제만 공부하고 다른 사람의 주제에 대해 질문을 하지 못하는 점이 아쉽습니다.
- 아직 이전의 내용들을 완벽히 이해를 하지 못하고 넘어가는 것 같아 저 개인적으로 복습의 시간이 필요할 것 같네요 😢

### 앞으로 어떻게 하면 좋을지
- (지금처럼) 처음보는 사람들을 위해 예제와 설명을 동시에 작성하는 방식으로 진행했으면 좋겠네요. 😋
- 추가로 공부하면서 알아가게된 점들이나 작은 팁(?)들도 부록으로 넣어보면 어떨까 싶네요.

**Comment by Ssong-Q** [comment]
### 스터디 하면서 좋았던 점
- 바쁜 와중에 내가 담당한 챕터 외에 다른 챕터를 읽지 못한 경우, 스터디를 통해 직접 해당 챕터를 학습한 분의 언어로 들을 수 있어서 좋았습니다.
- 혼자 였다면 초반 부분에서 읽다가 포기했을 것 같은데, 억지로라도 지속가능한 형태로 어쨌든 계속해서 나아가고 있다는 점이 좋습니다.
- 스터디 시간을 통해 꿀팁 등을 들을 수 있어서 좋았습니다.

### 아쉬웠던 점
- 어느 순간부터 내가 담당한 분량 외에는 많은 부분들을 놓치고 가고 있어서, 복습의 시간이 필요한 것 같습니다. 
- 중반부분을 지나면서 느낀 개인적인 느낌으로는 이펙티브 타입스크립트의 예시들을 읽고 이해하는데 어려움을 느끼고 있다는 것인데, 기본기 학습이 더 필요하다고 느낍니다.

### 앞으로 어떻게 하면 좋을지
- 프로젝트 기간이라 시간이 많이 없을 것 같아서 걱정인데, 다른 분들 요약본이라도 읽고 참석할 수 있도록 해야겠습니다.

**Comment by han-kimm** [comment]
## 스터디 하면서 좋았던 점
- 함께 하고 있다는 것에 대한 즐거움, 안정감.

## 아쉬웠던 점
- 읽은 내용에 대한 상호작용이 부족하다.
서로의 발표 들으면서 일방적인 전달에 그치지 않았나 반성합니다.

## 앞으로 어떻게 하면 좋을지
- 해당 주차마다 마니또를 지정해서, 그 사람에게 질문 하나 하기
